wp-content/themes/engage/assets/scss/components/_widget.scss

```diff
@@ -29,34 +29,59 @@
 		color: $black-grey;
 	}
 
-	.constantcontactwidget_form {
-		p {
-			margin-bottom: 1rem;
-			color: $black-grey;
-		}
+	.textwidget {
+		.wpcf7-form {
+			margin: 0;
 
-		input[type="text"] {
-			font: inherit;
-			margin-top: 1rem;
-			width: 100%;
+			.newsletter-prompt {
+				margin-bottom: 1.6rem;
+				color: $black-grey;
+			}
 
-			@include media($tablet) {
+			.newsletter-input {
+				font: inherit;
 				width: 100%;
+				border: 1px solid;
+
+				@include media($tablet) {
+					width: 100%;
```

Comment:
`width: 100%;` is already on this. Does it need to be here again in the media query to override something?